[IMP] pos: update customer display page (#17370)
odoo/documentation: content/applications/sales/point_of_sale/hardware_network/customer_display.rst

Customer display
================

The customer display feature provides real-time updates on a secondary screen for customers
during the checkout process. It shows the following information:

- :ref:`Items in the cart <pos/use/sell>`
- Unit :doc:`prices <../../point_of_sale/extra/pricing>`

> - Subtotals and total amount

Reviewer: ```suggestion
- Line (product) totals and overall total
```
to make it a bit clearer

- :ref:`Taxes <pos/pricing/taxes>`
- :ref:`Discounts and promotions <pos/pricing/loyalty>`
- Added :ref:`notes <pos/use/notes>`
- Selected :doc:`payment method <../payment_methods>`

> - Change to be returned

Reviewer: ```suggestion
- Change due, if applicable
```

.. example::
   On the customer display, the customer can review their order and all amounts, including discounts
   and taxes:

   - 2 pizzas at `19.36€` each
   - An `Extra spicy` burger at `18.76€`
   - A 10% discount applied to the order
   - `8.98€` in taxes

>    Finally, the customer pays `60.00€` and receives `8.27€` in change.

Reviewer: ```suggestion
   Finally, the customer pays `60.00€` in cash and receives `8.27€` in change.
```
so we mention the payment method
Great idea to have an example box here with the info listed along with the screenshot though!

   .. image:: customer_display/customer-display-example.png
      :scale: 70%
      :alt: Customer display view.

.. _pos/hardware_network/display-configuration:

Configuration
=============

The customer display can be shown on either a secondary screen connected via USB-C or HDMI, or on a

> screen connected through an :ref:`IoT box <pos/hardware_network/display-configuration-iot>`.

Reviewer: I think we also need to briefly mention what you explain on line 83-84 (QR code) here, don't we? Even though it doesn't require any configuration

but then do you you know a specific use case where this customer display QR feature would actually be useful? (I doubt people would buy devices with access to the Internet just to use them as a customer display? ... 🙈 )

Author: Yes, I will add the information. I guess you could also use a tablet and scan the QR code instead of using an IoT Box.

The customer display feature is built in, but the background image can be customized. To do so, go
to the :ref:`POS settings <pos/use/settings>` and scroll down to the :guilabel:`Connected Devices`
section. Under :guilabel:`Customer Display`, click :guilabel:`Upload your file` to set a

> background image.

Reviewer: - I don't think we need to mention that it's built-in
- I would rephrase slightly to avoid repetitions (*the background image can be customized* >< *to set a background image*)
- I would add another sentence for the file selection + saving of the settings
```suggestion
The background image can be customized in the :ref:`POS settings <pos/use/settings>. To do so, scroll down to the :guilabel:`Connected Devices` section and, under :guilabel:`Customer Display`, click :guilabel:`Upload your file`. Then, select the relevant image and click :guilabel:`Save`.
```
and maybe I would add a small tip after this to mention the fact that you can change the image by clicking the pencil icon? or do you think that's not necessary?

Author: Sure, why not 😄

.. note::
   Both the customer and POS displays must have a minimum diagonal size of 6 inches. For optimal
   readability, larger screens are recommended.

.. _pos/hardware_network/display-configuration-iot:

Using an IoT box
----------------

> To connect a display via an :doc:`IoT system </applications/general/iot>`:

Reviewer: Here I think we can use IoT box since it wouldn't make sense for Windows virtual IoT

```suggestion
To connect a display via an :doc:`IoT box </applications/general/iot/iot_box>`:
```

#. Navigate to the :ref:`POS settings <pos/use/settings>`.
#. Scroll down to the :guilabel:`Connected Devices` section.
#. Enable the :guilabel:`IoT Box` option.
#. Click :guilabel:`Save` to install the IoT app in Odoo (if not already installed).
#. :doc:`Connect and configure an IoT system </applications/general/iot/connect>` for the
   :doc:`screen </applications/general/iot/devices/screen>`.
#. Return to the :guilabel:`Connected Devices` section and select an IoT-connected screen in the

>    :guilabel:`Customer Display` field for the :guilabel:`IoT Box`.

Reviewer: I'm a bit on the fence here because we have [this doc](https://www.odoo.com/documentation/master/applications/sales/point_of_sale/hardware_network/pos_iot.html) that explains the different steps specific to connecting a POS with an IoT system.
And then if you connect the IoT box to the POS during the setup (see step 5 on the linked doc page), you don't need to do anything else, the devices are automatically linked to the POS. 
However (and that's what's mentionned in the tip on that doc page), if you want to connect a customer display (or another device for that matter) *after* the IoT system has already been setup, then you're going to have to go through the POS settings. But then the IoT Box setting will probably be enabled already I assume. So yes I think we should probably remove the info about what is strictly IoT box/system (and its connection to the db).

Here's my suggestion (to be discussed):
```suggestion
To connect a customer display via an :doc:`IoT box </applications/general/iot/iot_box/>`:

- either :doc:`connect the screen </applications/general/iot/devices/screen>` to the IoT box during :doc:`its setup <pos_iot>`;
- or navigate to the :ref:`POS settings <pos/use/settings>`, scroll down to the :guilabel:`Connected Devices` section, make sure the :guilabel:`IoT Box` setting is enabled, and select the relevant device in the :guilabel:`Customer Display` field.
```
What do you think?

Author: Good catch!!  You are totally right! I like your suggestion.

.. tip::

>    This configuration is especially useful for connecting tablets.

Reviewer: I'd clarify *this configuration* here following the suggested change above, and I would aslo change the part about *connecting tablets* I think it could be a bit confusing:

```suggestion
   Connecting a customer display via an IoT box is especially useful for tablet POS devices.
```
or something similar

.. _pos/hardware_network/open-display:

Opening the customer display
============================

To open the customer display, follow these steps:

#. :ref:`Access the POS register <pos/use/open-register>`.
#. Click the :icon:`fa-bars` (:guilabel:`hamburger menu`) icon in the upper-right corner.
#. Click the :icon:`fa-desktop` (:guilabel:`Customer Display`) icon.
#. On the :guilabel:`Open Customer Display` pop-up, choose the device to use for the customer
   display:

   - :guilabel:`This device`: Use this option if a secondary screen is connected via USB-C or
     HDMI, then drag the customer display window to that screen.
   - :guilabel:`Display QR`: Scan the QR code to open the customer display on any internet-connected
     device.

.. note::

>    For IoT-connected screens, the customer display opens automatically on the connected device. Make

Reviewer: ```suggestion
   For IoT-connected screens, the customer display opens automatically on the :ref:`connected device <pos/hardware_network/display-configuration-iot>`. Make
```

>    sure both devices are connected to the same :doc:`local network <pos_lna>`.

Reviewer: I'm not sure how relevant referring to the LNA doc is here, I think just saying network would be enough in this context? (but I might be wrong! let's discuss this if I am!):

```suggestion
   sure both devices are connected to the same network.
```

.. tip::
   For POS terminals running the
   `Odoo Android app <https://play.google.com/store/apps/details?id=com.odoo.mobile>`_ with
   dual screens, :doc:`activate the Point of Sale Mobile module <../../../general/apps_modules>`.
   To use the customer display, :ref:`open it <pos/hardware_network/open-display>`, and

>    :guilabel:`Select a display`.

Reviewer: I couldn't test this, where is the *Select a display* field exactly?

Author: I downloaded the app and tried it on my phone. So when I tried to open the customer display, a "Select a display" pop-up appeared. BUT there was no display to choose because I didn't install it on a second device.

Reviewer: ooooh okay, thank you for the explanation! that makes sense :slightly_smiling_face:
